cmd/snap/cmd_snap_op.go

```diff
@@ -919,6 +941,7 @@ func init() {
 		waitDescs.also(channelDescs).also(modeDescs).also(map[string]string{
 			"revision":          i18n.G("Refresh to the given revision"),
 			"list":              i18n.G("Show available snaps for refresh"),
+			"time":              i18n.G("Show auto refresh information"),
```

Comment:
(I see that `list` has the same problem of not saying “this will not perform the action”)